Makefile

```diff
@@ -521,29 +537,36 @@ endif
 
 .PHONY: .combine-integration-tests-coverage
 .combine-integration-tests-coverage: .run-integration-tests-coverage
-	. $(VIRTUALENV_DIR)/bin/activate; COVERAGE_FILE=.coverage.integration \
-	    coverage combine .coverage.integration.*
+	@if [ -n "$(NOSE_COVERAGE_FLAGS)" ]; then \
+	    . $(VIRTUALENV_DIR)/bin/activate; COVERAGE_FILE=.coverage.integration \
+	        coverage combine .coverage.integration.*; \
```

Comment:
BTW, combine looks suspicious, because `codecov` already combines everything natively: https://docs.codecov.io/docs/merging-reports

Reply:
That's true, but I also wanted to combine the tests locally. I can remove that if we want to.